terraform/newsletter_lambda.tf

```diff
@@ -0,0 +1,116 @@
+
+# lambda repo clone. 
+resource "null_resource" "aws_lambda_repo_clone" {
+  provisioner "local-exec" {
+    command     = <<-EOT
+    git clone ${var.github_repo}
+    pip3 install -r ${var.temp_path}/${var.foldername}/${var.github_repo_name}/requirements.txt -t ${var.temp_path}/${var.foldername}/${var.github_repo_name}/
```

Comment:
We probably need to inform the user as to what python version they need to use in order to install these requirements locally. Saying because it failed when i tried to run on Python 3.6